packages/graphql/src/translate/authorization/utils/filter-by-values.ts

```diff
@@ -55,3 +62,15 @@ export function filterByValues<T>(
     }
     return true;
 }
+
+function expandJwtClaim<T>(mappedJwtClaim: string, jwtData: Record<string, T>): T | undefined {
+    let receivedValue: T | undefined;
+    let paths = mappedJwtClaim.split(/(?<!\\)\./);
+
+    paths = paths.map((p) => p.replaceAll(/\\\./g, "."));
+
+    for (const p of paths) {
+        receivedValue = (receivedValue || jwtData)[p];
+    }
```

Comment:
I'm not sure anymore, but I think we used to have an util/lib for this kind of path access operations :thinking:

Reply:
Yeah, why not using `dot-prop` here?

Reply:
oh, yes you're right 🤦‍♀️